src/share/classes/java/sql/DriverManager.java

```diff
@@ -32,6 +32,8 @@
 import java.util.concurrent.CopyOnWriteArrayList;
```

Comment:
make all import package 'java.util.*' together

Reply:
better not